turnitintooltwo_view.class.php

```diff
@@ -715,7 +715,6 @@ private function get_submission_inbox_part_details($cm, $turnitintooltwoassignme
             $hidetext = html_writer::tag('i', '', array('class' => 'fa fa-minus-circle red fa-lg '.$hideclass));
             $showclass = 'show_summary_'.$turnitintooltwoassignment->turnitintooltwo->id;
             $showtext = html_writer::tag('i', '', array('class' => 'fa fa-plus-circle green fa-lg '.$showclass));
-            $links = html_writer::link('javascript:void(0)', $showtext.$hidetext , array('class' => 'toggle_summary'));
```

Comment:
`$links` is now always going to be an empty string, so you can just remove it completely

Reply:
I can remove the whole `if block`, however removing `$links` throws undeclared warnings because it is used later on in the code. Ex: Line 733

Reply:
@MartinLenord It's been removed completely. Can I get another look?